container/apptabs.go

```diff
@@ -89,6 +88,34 @@ func (t *AppTabs) CurrentTabIndex() int {
 	return t.current
 }
 
+// EnableItem enables the specified TabItem.
```

Comment:
We try to keep stuff alphabetical, please put enable after disable.
https://github.com/fyne-io/fyne/wiki/Contributing#code-style

Reply:
This is fixed now. My apologies for the delay!